.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+*Note: Please write your issue only in english*
+
+**Description**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone (please complete the following information):**
+
+- Device: [e.g. Samsung S20 Ultra 5G]
+- Android version: [e.g. Android 11]
+- App version: [e.g. 4.0.1]
```

Comment:
Maybe we could write 1.0.1 as example, to not throw off people, because we're far from version 4.0